core/files/css/global-css.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// TODO: Delete this file in v3.26.0 - It is not in use anymore.
```

Comment:
Why in version 3.26.0? What is the logic around that?

Reply:
@hein-obox 

```
'Fatal error: Uncaught Error: Class "Elementor\\Core\\Files\\CSS\\Global_CSS" not found in /var/www/html/wp-content/plugins/elementor/core/files/manager.php:118\n' +
1133
      'Stack trace:\n' +
```

I get this error when upgrading the plugin. It was flagged in the Playwright Upgrade test, and I reproduced it manually as well.

It is some weird WordPress issue that it still runs the previous version's `core/files/manager.php`, which looks for `Global_CSS` that is not there anymore. So I decided to keep the Global_CSS file for a few versions so that a version upgrade does not trigger a fatal error.

Reply:
You will see in this PR I removed all references to `Global_CSS` in `core/files/manager.php`, but still I get the above error, so somehow the old `manager.php` is still run in that process.

Reply:
@louiswol94 Thank you for the clarification.

Reply:
@louiswol94 Could you add this kind of reference as well: https://github.com/elementor/elementor/blob/f0c1718d59c8b07a1e09175f4d6e5c98c17e1b63/core/page-assets/loader.php#L39. 

And add this to do to this task: https://elementor.atlassian.net/browse/ED-15471.

We have more parts of the code that need to be updated in version 3.26.0.

Reply:
Done